src/diagnostic-channel-publishers/src/mysql.pub.ts

```diff
@@ -101,7 +101,7 @@ const mysqlPatchFunction: PatchFunction = function(originalMysql, originalMysqlP
 };
 
 export const mysql: IModulePatcher = {
-    versionSpecifier: ">= 2.0.0 <= 2.14.0",
+    versionSpecifier: ">= 2.0.0 <= 2.14.x",

```

Comment:
Glimpse had some infrastructure to test allow us to run tests against n different versions of a module.  We should consider pulling this over & leveraging it to validate against supported modules. 

Also, we had tests that would check to see that the latest version of a module wsa supported, and we would fail the test if not.  This let us know whenever we needed to update our semver ranges.